javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,21 @@
+var fullName;
+var isCool;
+var totalPets;
+var unknown;
+var never;
+
+fullName = 'Lamberto Martinez';
+isCool = false;
+totalPets = 0;
+unknown = null;
```

Comment:
Instead of assigning your variables down here, I would assign them when you initially declare them.

Reply:
downvote